modules/elasticsearch-index/src/main/java/org/opencastproject/elasticsearch/index/objects/event/EventQueryBuilder.java

```diff
@@ -192,8 +209,11 @@ public void buildQuery(EventSearchQuery query) {
     }
 
     // Recording start date
-    if (query.getStartDate() != null) {
-      and(EventIndexSchema.START_DATE, query.getStartDate());
+    if (query.getStartDateValues() != null) {
+      for (EventSearchQueryField<String> date : query.getStartDate()) {
+        addToQuery(EventIndexSchema.START_DATE,
+            date.getValue(), date.getType(), date.isMust());
+      }
```

Comment:
Where is the "startDate" coming from that is set in elastic search? Is from the Mediapackge's created or dcCreated values? Or does it come from the event's initial processing workflow?

Reply:
Where the dates are coming from was not something I was concerning myself with. The goal here was to extend the querying capabilities, particularly if a given date as older/earlier than another.